data_safe_haven/commands/cli.py

```diff
@@ -1,13 +1,12 @@
 """Command line entrypoint for Data Safe Haven application"""
 
-import pathlib
 from typing import Annotated, Optional
 
 import typer
+from rich import print as rprint
```

Comment:
I think we should replace this with a `console.print` function that passes its args through to `rich.print`. That would simplify our include structure and unify the things that write straight to the console.

Reply:
Do you mean [`rich.console.Console.print`](https://rich.readthedocs.io/en/stable/reference/console.html#rich.console.Console.print) or making something like `data_safe_haven.console.print`?

That could work with the other two comments here, introducing a `data_safe_haven.{console,output,terminal,text}` module to hold these things.